src/formats/base_32.cpp

```diff
@@ -86,22 +88,32 @@ data_chunk expand(const std::string& prefix)
 // Do the checksum math.
 uint32_t polymod(const data_chunk& values)
 {
-    static const uint32_t magic_numbers[] =
+    // Polynomials in the bech32 implementation are represented by
+    // simple integers. Generally 30-bit integers are used, where each
+    // bit corresponds to one coefficient of the polynomial.
+    //
+    // https://bitcoin.stackexchange.com/questions/74573/how-is-bech32-based-on-bch-codes
```

Comment:
Drop "https://" to shorten line and eliminate hyperlink highlighting in VS (which is annoying).